lib/samples/create_mobile_geodatabase/create_mobile_geodatabase_sample.dart

```diff
@@ -0,0 +1,357 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'dart:io';
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:share_plus/share_plus.dart';
+
+import '../../utils/sample_state_support.dart';
+
+class CreateMobileGeodatabaseSample extends StatefulWidget {
+  const CreateMobileGeodatabaseSample({super.key});
+
+  @override
+  State<CreateMobileGeodatabaseSample> createState() =>
+      _CreateMobileGeodatabaseSampleState();
+}
+
+class _CreateMobileGeodatabaseSampleState
+    extends State<CreateMobileGeodatabaseSample> with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // Declare a map to be loaded later.
+  late final ArcGISMap _map;
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // A mobile Geodatabase to be created and shared.
+  Geodatabase? _geodatabase;
+  // A feature table to store the location history.
+  GeodatabaseFeatureTable? _featureTable;
+  // A counter to keep track of the number of features added.
+  var _featureCount = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  child: Stack(
+                    children: [
+                      // Add a map view to the widget tree and set a controller.
+                      ArcGISMapView(
+                        controllerProvider: () => _mapViewController,
+                        onMapViewReady: onMapViewReady,
+                        onTap: _ready ? onTap : null,
+                      ),
+                    ],
+                  ),
+                ),
+                // Display the number of features added and a button to view the table.
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(
+                        'Number of features added: $_featureCount',
+                      ),
+                      ElevatedButton(
+                        style: OutlinedButton.styleFrom(
+                          shape: const RoundedRectangleBorder(
+                            borderRadius: BorderRadius.zero,
+                          ),
+                          side: const BorderSide(
+                            color: Colors.grey,
+                          ),
+                        ),
+                        onPressed: _displayTable,
+                        child: const Text(
+                          'View table',
+                        ),
```

Comment:
I think this button should be on the same row as the other button and be the same style.

Reply:
Based on the [design](https://devtopia.esri.com/runtime/common-samples/tree/main/designs/CreateMobileGeodatabase), the two buttons are in separate rows and different styles.

Reply:
Ok I will back down on that, I was looking at the Swift sample viewer but you're right the design is mobile and doesn't have it on the same row. That said, it should still match our sample viewer ElevatedButton style i.e. without extra border styling.

Reply:
updated